Portage/GHCCore.hs

```diff
@@ -44,6 +44,7 @@ cabalFromGHC ver = lookup ver table
           , ([7,10,2], Cabal.mkVersion [1,22,4,0])
           , ([8,0,1],  Cabal.mkVersion [1,24,0,0])
           , ([8,0,2],  Cabal.mkVersion [1,24,2,0])
+          , ([8,2,1],  Cabal.mkVersion [2,0,0,0])
```

Comment:
`ghc-8.2.1` comes with `2.0.0.2` specifically, not `2.0.0.0`:
```
$ ghc-pkg list Cabal
/usr/lib64/ghc-8.2.1/package.conf.d
    Cabal-2.0.0.2
```
You can also check `ls -1 /usr/lib/ghc-8.2.1/package.conf.d.initial/` for all the bundled versions.